src/cpu/pred/BranchPredictor.py

```
@@ -1206,3 +1206,7 @@ class DecoupledBPUWithBTB(BranchPredictor):
 
     bpDBSwitches = VectorParam.String([], "Enable which traces in the form of database")
     resolveBlockThreshold = Param.Unsigned(8, "Consecutive resolve dequeue failures before blocking prediction once")
+    enableFullResolveTrain = Param.Bool(Parent.enableFullResolveTrain,
```

Ruff (0.15.7)

[error] `Param` may be undefined, or defined from star imports

(F405)